CoreScriptsRoot/LoadingScript.lua

```diff
@@ -25,7 +25,8 @@ local function getViewportSize()
 		game.Workspace.Changed:wait()
 	end
 
-	while game.Workspace.CurrentCamera.ViewportSize == Vector2.new(0,0) do
+	while game.Workspace.CurrentCamera.ViewportSize == Vector2.new(0,0) or
```

Comment:
Leave a comment pointing to the line in cpp/lua where the initial viewport is set to a size.